python/cudf/cudf/tests/test_series.py

```diff
@@ -1346,3 +1346,35 @@ def test_nullable_bool_dtype_series(data, bool_dtype):
     gsr = cudf.Series(data, dtype=bool_dtype)
 
     assert_eq(psr, gsr.to_pandas(nullable=True))
+
+
+@pytest.mark.parametrize(
+    "data",
+    [
+        ([0, 1, 2, 3]),
+        (
+            [
+                "abc",
+                "a",
+                None,
+                "hello world",
+                "foo buzz",
+                "",
+                None,
+                "rapids ai",
+            ]
+        ),
+    ],
```

Comment:
```suggestion
    "data",
    [
        [0, 1, 2, 3],
        [
            "abc",
            "a",
            None,
            "hello world",
            "foo buzz",
            "",
            None,
            "rapids ai",
        ],
    ],
```

This could be a simple list too right?

Reply:
Yeah this works as well, thanks for pointing this out!